### ISSUE TITLE
`Enhancement`  - Add meta tag for latest release

### ISSUE THREAD

**Issue by thenoid** [Blackymas/NSPanel_HA_Blueprint#1740]
### Enhancement Summary

Would it be possible to get a "stable" or "latest-release" git tag added pointing 

### Detailed Description

Would it be possible to get a "stable" or "latest-release" git tag added pointing to the latest release.

### Additional Context

Updating this requires multiple steps, updating the yaml/restarting HA, updating the tft, updating ESPhome, hoping to write some automation/scripts and if there was a single tag pointing to the "stable" release would make my life easier.  If not super fine.

**Comment by edwardtfn** [comment]
I'm not sure if I got your request... We always flag the "latest-release" tag, and the code on main is always the latest (all the development happens in `dev` branch)...

**Comment by thenoid** [comment]
These are the current git tags: 
```root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git tag
latest
v.2.2.0
v.3.0.0
v.3.1.0
v.3.2
v.3.3
v2.0.0
v2.0.1
v2.0.2
v2.1.0
v3.2.1
v3.2.2
v3.4
v3.4.1
v4.0
v4.0.1
v4.0.2
v4.1
v4.1.1
v4.1.2
v4.1.3
v4.1.4
v4.2
v4.2.1
v4.2.2
v4.2.3
v4.2.4
root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git checkout latest
HEAD is now at c4b73e4 v1.0.1 - Fix Code and add Date Sensor to Blueprint
root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git rev-parse HEAD
c4b73e46f4702d63f0df30188d41688337cba24a

root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git checkout v4.2.4
Updating files: 100% (366/366), done.
Previous HEAD position was c4b73e4 v1.0.1 - Fix Code and add Date Sensor to Blueprint
HEAD is now at d704de8 Merge pull request #1681 from Blackymas/beta
root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git rev-parse HEAD
d704de853a029197a2ca8fe9c8466edef0c277d8
```

So where `v4.2.4` is the latest release, also have a "metatag" that points to the latest versioned  release eg: 
```
root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git tag stable
root@donnager:/srv/HA/repos/NSPanel_HA_Blueprint# git rev-parse HEAD
d704de853a029197a2ca8fe9c8466edef0c277d8
```

**Comment by edwardtfn** [comment]
This is right, v4.2.4 is the latest stale release.

**Comment by edwardtfn** [comment]
Could you please check if the tag is set now?

**Comment by edwardtfn** [comment]
We just released a patch with solves this: [v4.2.5 - Celebrating 1000 Stars with Enhanced Functionality and UI Improvements](https://github.com/Blackymas/NSPanel_HA_Blueprint/releases/tag/v4.2.5)
Could you please update your system and let me know if the issue was solved?
